test/src/Limit/LoggingLimitTest.php

```diff
@@ -7,9 +7,15 @@
 use PHPUnit\Framework\TestCase;
 use Psr\Log\LoggerInterface;
 
+/**
+ * @covers \eLife\Bus\Limit\LoggingLimit
```

Comment:
we don't calculate coverage (doing so usually makes the build much slower by requiring xdebug), but there's no harm in the annotation